docs/tempo/website/community/_index.md

```diff
@@ -3,7 +3,13 @@ title: Community
 weight: 600
 ---
 
-# Contribute
+## Communicate
+
+- [Grafana Slack](https://slack.grafana.com/) #tempo channel
+- [Grafana Tempo Community Forum](https://community.grafana.com/c/grafana-tempo/40)
+- [Google Groups](https://groups.google.com/forum/#!forum/tempo-users)
```

Comment:
Add something in here about the community call please. 

Link to the google doc: https://docs.google.com/document/d/1yGsI6ywU-PxZBjmq3p3vAXr9g5yBXSDk4NU8LGo8qeY/edit#